src/common/types/SmtpNotificationData.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: Update and define SmtpNotificationData type
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type SmtpNotificationData = any;
```

Comment:
This is not needed.

Reply:
This has been deleted in the latest commit